spec/controllers/idv/in_person/state_id_controller_spec.rb

```diff
@@ -115,7 +115,12 @@
   describe '#update' do
     let(:first_name) { 'Natalya' }
     let(:last_name) { 'Rostova' }
-    let(:dob) { InPersonHelper::GOOD_DOB }
+    let(:formatted_dob) { InPersonHelper::GOOD_DOB }
+    let(:dob) do
+      { month: formatted_dob.split('-')[1],
+        day: formatted_dob.split('-')[2],
+        year: formatted_dob.split('-')[0] }
+    end
```

Comment:
we're not using this whole hash, I wouldn't bother defining it
```suggestion
```

Reply:
This hash is needed because the `Form.submit` isn't mocked out in the tests, which requires hash version of the dob to be in the params/flow_params.

The reason I updated the `dob` in the tests was because the params passed to the controller are the parsed version. When I was debugging the application I noticed this and saw that the tests didn't reflect what is actually being sent. Was this the right change? It looks like at some point the past this wasn't the case.